<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>750</width>
     <height>850</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>850</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0" colspan="4">
       <widget class="QLabel" name="spectogramLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>750</width>
          <height>350</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QLabel" name="audioDurationLabel">
          <property name="text">
           <string>Czas: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="chooseImgFile">
          <property name="text">
           <string>Wybierz obraz do ukrycia</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="heightLabel">
          <property name="text">
           <string>wys = </string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="minLumLabel">
          <property name="text">
           <string>min. lum = </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>dł = </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="selectAudioFileButton">
          <property name="text">
           <string>Wybierz plik audio</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="samplingLabel">
          <property name="text">
           <string>Próbkowanie: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLabel" name="imgFileNameLabel">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Obraz: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="audioFileNameLabel">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Audio:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="maxLumLabel">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>13</height>
           </size>
          </property>
          <property name="text">
           <string>max. lum = </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="1" colspan="2">
         <widget class="QLabel" name="constLabel4">
          <property name="text">
           <string>klatek częstotliwościowych</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="4">
         <widget class="QLabel" name="totalFreqFramesLabel">
          <property name="text">
           <string>klatek częst = </string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" colspan="4">
         <widget class="QLabel" name="calculatedFreqToStart">
          <property name="text">
           <string>= 0 Hz</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="startImgSpinBox_T">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLabel" name="constLabel3">
          <property name="text">
           <string>klatek czasowych</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QSpinBox" name="imgDurationSpinBox_F">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="totalTimeFramesLabel">
          <property name="text">
           <string>klatek czasowych = </string>
          </property>
         </widget>
        </item>
        <item row="8" column="3" colspan="3">
         <widget class="QLabel" name="calculatedTimeToUse">
          <property name="text">
           <string>= 0 ms</string>
          </property>
         </widget>
        </item>
        <item row="8" column="6">
         <widget class="QLabel" name="WToHRatioLabel_org">
          <property name="text">
           <string>Oryginalny stosunek W/H : </string>
          </property>
         </widget>
        </item>
        <item row="9" column="6">
         <widget class="QLabel" name="WToHRatioLabel_mod">
          <property name="text">
           <string>Prszekalowany stosunek W/H :        </string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="4">
         <widget class="QLabel" name="calculatedTimeToStart">
          <property name="text">
           <string>= 0 ms</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="7">
         <widget class="QPushButton" name="calculateSTFTButton">
          <property name="text">
           <string>Oblicz spektogram obrazu oryginalnego obrazu</string>
          </property>
         </widget>
        </item>
        <item row="9" column="3" colspan="3">
         <widget class="QLabel" name="calculatedFreqToUse">
          <property name="text">
           <string>= 0 Hz</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <widget class="QLabel" name="constLabel1">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>klatek czasowych</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="2">
         <widget class="QLabel" name="constLabel2">
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>klatek częstotliwościowych</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QSpinBox" name="startImgSpinBox_F">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="7">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="resetSpectLabelButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Rysuj czysty spektogram</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="setImgInSpectButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Umieść obraz w spektogramie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveResultsButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Zapis wyniki</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <widget class="QSpinBox" name="imgDurationSpinBox_T">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0" rowspan="3" colspan="7">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="scalingChekbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Skaluj obraz</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="amplifierSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>0</height>
             </size>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-40.000000000000000</double>
            </property>
            <property name="maximum">
             <double>-10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.200000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="constLabel5">
            <property name="text">
             <string>wzmocneinei[dB]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="4">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectAudioFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseAudioFileButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseImgFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseImageFileButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imgDurationSpinBox_F</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>frequencyDurationChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startImgSpinBox_F</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>startFrequencyChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startImgSpinBox_T</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>startTimeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imgDurationSpinBox_T</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>durationTimeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calculateSTFTButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calculateSTFT()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalingChekbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>allowScalingChechBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveResultsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveResultsToFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetSpectLabelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetLabelToUnmodifiedSpectrogram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setImgInSpectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addImgToTmpSpectrogram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amplifierSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>aplifierSpinBoxChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseAudioFileButtonPressed()</slot>
  <slot>chooseImageFileButtonPressed()</slot>
  <slot>startTimeChanged()</slot>
  <slot>startFrequencyChanged()</slot>
  <slot>durationTimeChanged()</slot>
  <slot>frequencyDurationChanged()</slot>
  <slot>calculateSTFT()</slot>
  <slot>allowScalingChechBoxChanged()</slot>
  <slot>resetLabelToUnmodifiedSpectrogram()</slot>
  <slot>addImgToTmpSpectrogram()</slot>
  <slot>saveResultsToFiles()</slot>
  <slot>aplifierSpinBoxChanged()</slot>
 </slots>
</ui>
